### PR TITLE
Made loading of blobs from database more robust

### PR DESCRIPTION
**Pull request by mpe85**
Fixes https://github.com/eclipse/jetty.project/issues/3890

### CODE REVIEW

File: jetty-server/src/main/java/org/eclipse/jetty/server/session/DatabaseAdaptor.java [hunk 1]
```diff
@@ -158,14 +158,14 @@ public InputStream getBlobInputStream(ResultSet result, String columnName)
         if (_dbName == null)
             throw new IllegalStateException("DbAdaptor missing metadata");
 
-        if (_dbName.startsWith("postgres"))
-        {
+        try {
+            Blob blob = result.getBlob(columnName);
+            return blob.getBinaryStream();
+        }
+        catch (SQLException ex) {
```
Comment (janbartel) [inline jetty-server/src/main/java/org/eclipse/jetty/server/session/DatabaseAdaptor.java]: I think this should catch "SQLFeatureNotSupportException", no?

File: jetty-server/src/main/java/org/eclipse/jetty/server/session/DatabaseAdaptor.java [hunk 2]
```diff
@@ -158,14 +158,14 @@ public InputStream getBlobInputStream(ResultSet result, String columnName)
         if (_dbName == null)
             throw new IllegalStateException("DbAdaptor missing metadata");
 
-        if (_dbName.startsWith("postgres"))
-        {
+        try {
```
Comment (janbartel) [inline jetty-server/src/main/java/org/eclipse/jetty/server/session/DatabaseAdaptor.java]: The jetty style is to have open and close curlies on new lines. See https://www.eclipse.org/jetty/documentation/9.4.19.v20190610/coding-standards.html

Comment (mpe85) [reply jetty-server/src/main/java/org/eclipse/jetty/server/session/DatabaseAdaptor.java]: Fixed

File: jetty-server/src/main/java/org/eclipse/jetty/server/session/DatabaseAdaptor.java (hunk 1 above)
Comment (mpe85) [reply jetty-server/src/main/java/org/eclipse/jetty/server/session/DatabaseAdaptor.java]: I was catching SQLException to also handle the 'postgres' case.
I changed it now to only catch  SQLFeatureNotSupportException and the 'postgres' case is now handled up-front as it was done before.

**Pull request merged**